src/scene/container/RenderGroupSystem.ts

```diff
@@ -161,7 +164,7 @@ export class RenderGroupSystem implements System
         }
         else if (renderGroup.texture)
         {
-            TexturePool.returnTexture(renderGroup.texture);
+            TexturePool.returnTexture(renderGroup.texture, true);
```

Comment:
hey @mayakwd there is one more place where `returnTexture` is called for this texture:

https://github.com/pixijs/pixijs/blob/7d91234eb506457fb6e612b02b31cdf6dae7062f/src/scene/container/RenderGroup.ts#L154